{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,111 +1,160 @@
+to see all the sushi chef scripts. They are all example of how to extract,
+transform, and upload content from various sources of openly licensed content.
+
+
+## Instructions and channel rubric
+A sushi chef script has been created for you in `sushichef.py` to help you get
+started on the import of the {{cookiecutter.channel_name}} content.
+
+1. Start by looking at the **channel spec** that describes the target channel structure,
+   licensing information, and tips about content transformations that might be necessary.
+2. Add the code necessary to create this channel by modifying the `construct_channel`
+   method of the chef class defined in `sushichef.py`.
+
+Use the following rubric as a checklist to know when your sushi chef script is done:
+
+### Main checks
+1. Does the channel correspond to the spec provided?
+2. Does the content render as expected when viewed in Kolibri?
+
+### Logistic checks
+1. Is the channel uploaded to Studio and PUBLISH-ed?
+2. Is the channel imported to a demo server where it can be previewed?
+3. Is the information about the channel token, the studio URL, and demo server URL
+   on notion card up to date? See the [Studio Channels table](https://www.notion.so/761249f8782c48289780d6693431d900).
+   If a card for your channel yet doesn't exist yet, you can create one using
+   the `[+ New]` button at the bottom of the table.
+
+### Metadata checks
+1. Do all nodes have appropriate titles?
+2. Do all nodes have appropriate descriptions (when available in the source)?
+3. Is the correct [language code](https://github.com/learningequality/le-utils/blob/master/le_utils/resources/languagelookup.json)
+   set on all nodes and files?
+4. Is the `license` field set to the correct value for all nodes?
+5. Is the `source_id` field set consistently for all content nodes?
+   Use unique identifiers based on the source website or permanent url paths.
+
+### Code standards
+1. Does the section `Usage` in this README contain all the required information
+   for another developer to run the script?
+   Document and extra credentials, env vars, and options needed to run the script.
+2. Is the Python code readable and succinct?
+3. Are clarifying comments provided where needed?
+
+
+## Kolibri content development workflow
+Running the sushichef script is only one of the steps in the Kolibri content
+development workflow:
+
+             ricecooker     studio       kolibri demo server
+    SPEC-->--CHEF----->-----PUBLISH--->--IMPORT using token and REVIEW
+    (1)      (2)            (3)          (4)                    (5)
+      \______/                                                 /
+       \______________________________________________________/
+
+It is your responsibility as the chef author to take this channel all the way
+through this workflow and make sure that the final channel works in Kolibri.
+
+Notes on specific steps:
+  - `(1)`: the spec for the channel describes how the channel should be organized
```

Comment:
I'd recommend using the actual step names instead of the number. I think it helps reinforce the terms. Also, I think it would be good to have some whitespace between each step to help keep the text for all the steps from blending together. (I know given I'm asking for this to be a link that these notes probably belong in the docs, but noting here so I don't forget!)

Reply:
Good idea. I removed the numbering and using step names instead (see ricecooker PR https://github.com/learningequality/ricecooker/pull/218/files )